### PR TITLE
Update 2.18

### PR DESCRIPTION
**Pull request by chuckcarpenter**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,15 +35,21 @@ matrix:
 
 before_install:
   - npm config set spin false
+<<<<<<< HEAD
```
Comment (ynx) [inline .travis.yml]: Eh?

File: .travis.yml [hunk 2]
```diff
@@ -26,15 +35,21 @@ matrix:
...
   - bower --version
   - phantomjs --version
 
 install:
   - npm install
   - bower install
+=======
+  - npm install -g npm@4
+  - npm --version
+>>>>>>> a63987b... message
```
Comment (ynx) [inline .travis.yml]: Duh?

File: addon/services/request-signer.js [hunk 3]
```diff
@@ -102,7 +99,14 @@ export default Service.extend({
    * @type {Array}
    * @public
    */
-  signedHeaders: [],
+  signedHeaders: null,
+
+  init() {
+    this._super(...arguments);
+
+
```
Comment (ynx) [inline addon/services/request-signer.js]: Too many new lines.

File: .travis.yml (hunk 2 above)
Comment (chuckcarpenter) [reply .travis.yml]: see why I get you?

**Pull request merged**